pkg/client/kubernetes/apply.go

```diff
@@ -136,6 +136,9 @@ var DefaultApplierOptions = ApplierOptions{
 			newObj.Object["secrets"] = oldObj.Object["secrets"]
 			newObj.Object["imagePullSecrets"] = oldObj.Object["imagePullSecrets"]
 		},
+		"VerticalPodAutoscaler": func(newObj, oldObj *unstructured.Unstructured) {
```

Comment:
I'd rather see this being [added from outside](https://github.com/gardener/gardener/blob/0.19.0/pkg/operation/seed/seed.go#L373) than being added directly to the `DefaultApplierOptions`.

Reply:
Thanks for the review. I moved it out